pkg/jaeger/tracer.go

```diff
@@ -205,14 +225,21 @@ func StartSpanIfParentSpanExist(req *restful.Request, operationName string) (ope
 	return span, opentracing.ContextWithSpan(req.Request.Context(), span)
 }
 
+func StartSpanSafe(operationName string, opts ...opentracing.StartSpanOption) opentracing.Span {
+	globalTracerAccessMutex.Lock()
+	defer globalTracerAccessMutex.Unlock()
+
+	return opentracing.StartSpan(operationName, opts...)
```

Comment:
are we sure this needs mutex? the lib is not threadsafe?

Reply:
@anggorodewanto correct. caught by https://github.com/AccelByte/go-restful-plugins/pull/30/files#diff-76ed074a9305c04054cdebb9e9aad2d818052b07091de1f20cad0bbac34ffb52R5